test/integration/features/basic.feature

```diff
@@ -102,15 +102,6 @@ Feature: Basic
       hello
       """
 
```

Comment:
@gbraad I am bit skeptical about it since atleast this steps making sure with `--docker-env` option minishift is up and running even we are not checking if those options present or not.